### PR TITLE
Rename spt references to schibsted

### PR DESCRIPTION
**Pull request by SimenB**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,9 +1,6 @@
 {
     "extends": [
-        "spt",
-        "spt/env-node"
-    ],
-    "globals": {
```
Comment (SimenB) [inline .eslintrc:6]: `env-node` did this better already, so dead code

Comment (alexewerlof) [reply .eslintrc:6]: I agree. I only use `env: node` and it's enough for my node server.

Comment (nosilleg) [reply .eslintrc:6]: This is a breaking change, but as the entire commit is, it's not an extra issue.  Should probably be mentioned in the changelog, tho.

**Comment by nosilleg** [comment]
I'm happy for this to go forward.  👍

File: .eslintrc (hunk 1 above)
Comment (SimenB) [reply .eslintrc:6]: Not breaking, it's this repo's config, not what's exported

**Comment by alexewerlof** [comment]
I have [an objection](https://github.com/schibsted/eslint-config-schibsted/pull/26#issuecomment-269104262) that came to light in #26.

Shall we address that before merging this?

File: packages/eslint-config-schibsted/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+```bash
+npm install --save-dev eslint-config-schibsted eslint-config-schibsted-react eslint-plugin-react@4
+```
+
+Then add the `extends` option to your `.eslintrc`:
+```json
+{
+    "extends": [
+        "schibsted",
+        "schibsted-react"
```
Comment (ivarconr) [inline packages/eslint-config-schibsted/README.md]: probably drop everything about react in this module as that's a separate module now.

**Review by ivarconr — APPROVED** [review]

File: packages/eslint-config-schibsted/README.md (hunk 2 above)
Comment (SimenB) [reply packages/eslint-config-schibsted/README.md]: https://github.com/schibsted/eslint-config-schibsted/commit/d47f61f3a60abbae2b59c357842a12753ca33b7a

**Review by sveisvei — APPROVED** [review]

**Comment by SimenB** [comment]
@userpixel This is OK (pending conflict resolution)?

**Comment by alexewerlof** [comment]
@SimenB all clear 👍

**Pull request merged**